articles/sso/current/introduction.md

```diff
@@ -49,22 +50,20 @@ Let's look at an example of how the SSO flow looks when using Auth0 and the [Loc
 
     ![](/media/articles/sso/single-sign-on/lock-no-sso-cookie.png)
 
-1. Once the user has logged in, Auth0 will set an SSO cookie
-1. Auth0 will also redirect back to your web application and will return an ID Token containing the identity of the user.
+1. Once the user has logged in, Auth0 will set an SSO cookie.
+1. Auth0 will also redirect back to your web application and will return an `id_token` containing the identity of the user.
```

Comment:
This is probably better how it was - I think we're standardizing the spelling/cap of `ID Token` and `Access Token` and attempting to use the name unless it's an actual code example.